README.tr.md

```diff
@@ -0,0 +1,57 @@
+<p align="center"><img src="https://user-images.githubusercontent.com/56656996/99621835-b2d8dc80-29dd-11eb-8183-987e80f8b3a7.png"></p>
+<h1 align="center">DDE Store</h1>
+<h3 align="center">DTK ile oluşturulmuş DDE için bir uygulama mağazası</h3>
+
+
+## Bu nedir?
+ 
+Bu uygulama mağazası, Deepin Masaüstü Ortamı'nın (Deepin Desktop Environment, DDE) diğer dağıtımlar üzerinde kullanılması amacıyla tasarlanmıştır. Resmi Deepin mağazası Debian ve üzerinde Deepin depolarının bulunduğu bir web görünümü kapsamında inşa edilmiştir, bu da Deepin mağazasını diğer dağıtımlarla uyumsuz hale getirir. Bu mağaza, PackageKit ve AppStream ile oluşturulmuştur, bu nedenle herhangi bir dağıtım üzerinde [almost](#deepin-distro) mükemmel çalışmalıdır!
+
+## Deepin Dağıtımı
+
+Bu sadece resmi Deepin dağıtımı için geçerlidir, genel olarak DDE için geçerli değildir. Deepin, bu uygulamaya gerekli tüm verileri sağlayan AppStream'i desteklemez. **Bu mağaza, Deepin dağıtımında desteklenmiyor.** Bu, yine de DDE'yi çalıştıran diğer dağıtımlar için tasarlandı, bu yüzden onu desteklemeye dair hiçbir planım yok. Deepin AppStream'i kullanmaya başlarsa, düşünebilirim. Deepin kullanıyor ve alternatif bir uygulama mağazası arıyorsanız, [Spark Store] (https://www.spark-app.store) veya [Deepines Store] (https: // deepines. com) gibi diğer harika projelere göz atın!

```

Comment:
Spark Store and Deepines Store links are broken. Links need to be formatted like this:
```markdown
[Deepines Store](https://deepines.com)
```
Make sure not to add any extra spaces between the brackets or in the link.

Reply:
Corrected with thanks.